src/CreepLaw/Data/DislocationCreep.jl

```diff
@@ -486,9 +486,9 @@ function wet_quartzite_Kirby_1983()
     #  referring to Koch et al. (1981), unpublished manuscript...
     data = DislocationCreep(;
         Name="Wet Quarzite | Kirby (1983)",
-        n=2.3NoUnits,
-        A=uconvert(Pa^(-23//10) / s, 3.2e-4MPa^(-23//10) / s),
-        E=154e3J / mol,
+        n=2.4NoUnits,
+        A=1.25e-2MPa^(-2.4), #A is given in 10^(3.2)^ GPa^(2.4)^ in paper
```

Comment:
```suggestion
        A=1.25e-2MPa^(-12//5), #A is given in 10^(3.2)^ GPa^(2.4)^ in paper
```
Float powers units are type unstable. Make them rational numbers, they are stable.